content/community/team.md

```diff
@@ -5,172 +5,9 @@ title: "Team"
 This page lists who we are. By all means add yourself to the list - lets sort it in alphabetical order
 
 # Committers
+{{< committers >}}
 
-When posting to the mailing lists, use plain text mails. Do not use HTML mails. HTML mails is more likely to be targeted as spam mails and will be rejected; as well it's not easily readable by others.
 
-{{< table >}}
-| Name  | ID | Organisation |
-|-------|----|--------------|
-| Aaron Mulder | ammulder | Chariot Solutions |
-| Akitoshi Yoshida | ay | SAP |
-| Andrea Cosentino | acosentino | Red Hat |
-| Antonin Stefanutti | astefanutti | Red Hat |
-| Babak Vahdat | bvahdat | Cyberlogic Consulting GmbH |
-| Ben O'Day | boday | Initek Consulting |
-| Bilgin Ibryam | bibryam | Red Hat |
-| Bruce Snyder | bsnyder | |
-| Charles Moulliard | cmoulliard | Red Hat
-| Christian Mueller | cmueller | Amazon Web Services |
-| Christian Posta | ceposta | Red Hat |
-| Christian Schneider | cschneider
-| Claus Ibsen | davsclaus | Red Hat |
-| Colm O hEigeartaigh | coheigea | Talend |
-| David Jencks | djencks | IBM |
-| Daniel Kulp | dkulp | Talend |
-| Denis Istomin | distomin | |
-| Dmitry Volodin | dmvolod | Red Hat |
-| Franz Forsthofer | forsthofer | SAP |
-| Freeman Fang | ffang | Red Hat |
-| Gary Tully | gtully | Red Hat |
-| Gert Vanthienen | gertv | |
-| Gregor Zurowski | gzurowski | Sotheby's |
-| Guillaume Nodet | gnodet | Red Hat |
-| Hadrian Zbarcea | hadrian | |
-| Henryk Konsek | hekonsek | |
-| Hiram Chirino | chirino | Red Hat |
-| Ioannis Canellos | iocanel | Red Hat |
-| Jacek Laskowski | jlaskowski | | 
-| James Carman | jcarman | Savoir Technologies |
-| Jan Bednar | janbednar | CGI IT Czech Republic |
-| Jean-Baptiste Onofré | jbonofre | Talend |
-| Jeff Genender | jgenender | Savoir Technologies |
-| Johan Edstrom | joed | Savoir Technologies |
-| John Poth | | Red Hat |
-| Jonathan Anstey | janstey | Red Hat |
-| James Netherton | Red Hat | |
-| James Strachan | jstrachan | |
-| Luca Burgazzoli | lburgazzoli | Red Hat |
-| Martin Krasser | krasserm | |
-| Nicky Sandhu | nsandhu | |
-| Onder Sezgin | onders | |
-| Omar Al-Safi | oalsafi | Talend |
-| Quinn Stevenson | quinn | |
-| Raul Kripalani | raulk | |
-| Pascal Schumacher | pascalschumacher | | 
-| Peter Palaga | ppalaga | Red Hat |
-| Rich Newcomb | rnewcomb | |
-| Richard Kettelerij | rickette | Avisi BV |
-| Rob Davies | rajdavies | |
-| Roman Kalukiewicz | romkal | |
-| Scott England-Sullivan | sully6768 | Red Hat |
-| Stan Lewis | slewis | Red Hat |
-| Tadayoshi Sato | tsato | Red Hat |
-| Tomohisa Igarashi | igarashitm | Red Hat |
-| Tracy Snell | tjsnell | Juice Labs |
-| Willem Jiang | ningjiang | Huawei |
-| William Tam | wtam | Red Hat |
-| Zoran Regvart | zregvart | Red Hat |
-{{< /table >}}
+# Contributors
```

Comment:
For some reason this is not rendered as a heading but literally (see preview)

Reply:
This is fixed with #265, make sure you rebase on that. I think the issue is with non breaking white space.